draft/spec/index.html

```diff
@@ -868,6 +868,28 @@ <h2>Extensions</h2>
             </p>
         </blockquote>
     </section>
+    
+    <section id="filesystem-features">
+        <h2>Filesystem features</h2>
+        <p>
+            In order to maximize the compatibility of the OCFL with different filesystems, and thus improve the
+            portability of OCFL Objects between different systems, some restrictions on the use of certain
+            filesystem features are necessary. If the preservation of non-OCFL-compliant features is required then
+            data MUST be wrapped in a suitable disk or filesystem image format which OCFL can treat as a regular
```

Comment:
'data' => 'the contents' (to make it more consistent with how we refer to the content of an OCFL object)

Reply:
or "the content" ? (To say "the contents" begs "of what?" to me)

Reply:
Sure; I just wanted to make sure we were being consistent, since we are trying to talk about both data and metadata as the 'content' of an OCFL Object.